### PR TITLE
use nav-hq-sidebar for app summary page

### PR DESCRIPTION
**Pull request by benrudolph**
@biyeun fixed up some bootstrap sidebar action on the app summary page
enchantress: @millerdev 
before:
screen shot 2015-04-28 at 8 26 23 pm
after:
screen shot 2015-04-28 at 8 26 07 pm

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html [hunk 1]
```diff
@@ -35,4 +35,4 @@
       </div>
   </div>
 </script>
-{% endangularjs %}
```
Comment (czue) [inline corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html]: i'm confused by this diff. i see the `{% endangularjs %}` in the file but it looks like it's getting (wrongly?) removed here. any idea what's happening?

Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html]: woah didn't notice that. it happened on the other file too. i'll update

Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html]: yea i actually think this is an issue with the diff? it exists on my branch locally

Comment (czue) [inline corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html]: super weird. github bug?

File: corehq/apps/style/static/style/less/components-3/navs.less [hunk 2]
```diff
@@ -12,13 +12,10 @@
       background-color: #dbdbdb;
     }
     &.active {
-      a, a:link, a:visited, a:hover {
+      > a, > a:link, > a:visited, > a:hover {
```
Comment (czue) [inline corehq/apps/style/static/style/less/components-3/navs.less]: I'm not 100% sure this is the right place to make this change so will let @biyeun take a look. g2g from me though

Comment (biyeun) [inline corehq/apps/style/static/style/less/components-3/navs.less]: :+1:

**Pull request merged**